### ISSUE TITLE
undefined is not an object ('_reactNative.ActivityIndicator.propTypes.color')

### ISSUE THREAD

**Issue by rahulbhankar786** [douglasjunior/react-native-simple-dialogs#23]
After upgrading to 0.57.2. getting this error.

```
undefined is not an object ('_reactNative.ActivityIndicator.propTypes.color')
```

"npm": "^6.4.1",
"react": "16.5.1",
"react-apollo": "^2.1.3",
"react-native": "^0.57.2",

**Comment by douglasjunior** [comment]
Hummm, I'll take a look

**Comment by douglasjunior** [comment]
Sorry, this has already been fixed in #17

**Comment by rahulbhankar786** [comment]
Thanks its working now.
